eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -22,9 +22,9 @@
   <!--  <suppress checks="com\.azure\.tools\.checkstyle\.checks\..+" files=".*[/\\]com[/\\]microsoft[/\\].*"/>-->
 
   <!-- ignore missing javadoc in samples -->
-  <suppress checks="MissingJavadocMethod" files=".*[/\\]samples[/\\].*\.java"/>
```

Comment:
Do we really want to unable these rules?

Reply:
yes eventually for all libs. 
Ref - https://github.com/Azure/azure-sdk-for-java/issues/9360

But package owners need to start at their own respective areas and then this suppression can finally be removed.

Reply:
I will do this for TA and MA too. jfyi